src/headless/converse-muc.js

```diff
@@ -507,6 +507,40 @@ converse.plugins.add('converse-muc', {
                     }, null, 'message', 'groupchat', null, room_jid,
                     {'matchBareFromJid': true}
                 );
+                this.muc_notifications_handler = _converse.connection.addHandler(stanza => {
+                    const item = converse_muc_sizzle("x[xmlns=\"".concat(converse_muc_Strophe.NS.MUC_USER, "\"] item"), stanza).pop();
```

Comment:
What is `converse_muc_sizzle`? I don't see it being defined anywhere.

Looks like you copied code that was generated by a  bundler or something?

Also, this selector string is difficult to parse:
`x[xmlns=\"".concat(converse_muc_Strophe.NS.MUC_USER, "\"] item"`

Rather use a [template literal](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals).

For example: 
    `x[xmlns="${Strophe.NS.MUC_USER}"], item`

Reply:
>Looks like you copied code that was generated by a bundler or something?

Sorry, I pasted the wrong code with my debugging console.log from converse.js. I shall redo